builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -90,6 +95,26 @@ node {
 		if (!service_config) {
 			error "Failed to fetch service metadata from catalog"
 		}
+		// Get the Account Details from the ConfigLoader
+		accountInfo = utilModule.getAccountInfo(service_config)
+
+    if(!service_config.region) {
+      service_config.region = configLoader.AWS.DEFAULTS.REGION
+    }
+
+    if(!service_config.accountId) {
+      service_config.accountId = configLoader.AWS.DEFAULTS.ACCOUNTID
+    }
+
+    if(!service_config.provider) {
+      service_config.provider = configLoader.JAZZ.DEFAULTS.PROVIDER
+    }
+
+		if(!service_config.credentialId){
```

Comment:
I don't know if this ever become true. Why would credentialId come from service catalog?

Reply:
removed